build/craft_release.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# This script is used to create a new release rock image from the included template
+set -eu
+
+SCRIPT_DIR="$( dirname "${BASH_SOURCE[0]}")"
+REPO_DIR=$(dirname ${SCRIPT_DIR})
+VERSIONS=${SCRIPT_DIR}/../versions.txt
```

Comment:
Maybe we can add a `readonly` here?
```suggestion
readonly SCRIPT_DIR="$( dirname "${BASH_SOURCE[0]}")"
readonly REPO_DIR=$(dirname ${SCRIPT_DIR})
readonly VERSIONS=${SCRIPT_DIR}/../versions.txt
```